console/src/main/webapp/manager/app/components/user/user.tpl.html

```diff
@@ -50,7 +50,7 @@ <h4>
         <h4 translate>role.system</h4>
         <div class="form-group clearfix">
           <div class="col-sm-6" ng-repeat="role in user.adminRoles" ng-if="user.allroles.indexOf(role) !== -1 && role !== 'ORGADMIN'">
-            <div class="checkbox">
+            <div class="checkbox" ng-repeat="r in user.systemRoles" ng-if="role === r.cn" title="{{r.description}}">
```

Comment:
This loop inside loop looks sub-optimal to me…

Reply:
What would you suggest ?

Reply:
In the callback pointed above, store role or description in a dict with `cn` as key to avoid another loop here.

Reply:
Thanks. That's was my first idea, i thought it was more clean to realize it into template. But your advice is more simple to read and effective.